app/covidbed/resource/orga.py

```diff
@@ -0,0 +1,72 @@
+
+import datetime
+
+from flask import request, jsonify
+from flask_jwt_extended import create_access_token, jwt_required
+from flask_restful import  Resource
+from flask_restful_swagger import swagger
+
+from covidbed.serializer.orga import OrganizationSerializer
+
+from covidbed.repository import orga as orga_repository
+from covidbed.validator import orga as orga_validator
+
+from sqlalchemy.orm.exc import NoResultFound
+
+class OrganizationAPI(Resource):
+    method_decorators = [jwt_required]
+
+    # @swagger.operation(
```

Comment:
Don't forget to uncomment the swagger annotation. 
Due to get parameters are path, you should describe each parameters as this:
 ```
parameters =  [
     {
                    "allowMultiple": false,
                    "dataType": "string",
                    "description": "The ID of the TODO item",
                    "name": "todo_id_x",
                    "paramType": "path",
                      "required": false
          }
        ]
```
You can define the response via a define serialisers: 

 ```
       responseClasse=OrganizationSerializer.__name__,
```